src/Authentication.Abstractions/Interfaces/IPSCmdletAction.cs

```diff
@@ -0,0 +1,23 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Azure.Commands.Common.Authentication.Abstractions
+{
+    public interface IPSCmdletAction
+    {
+        void LogTestCoverageRawData(string moduleName, string commandName, string parameterSetName, string parameters, string sourceScript, int scriptLineNumber, bool isSuccess);
```

Comment:
This method has a lot of parameters, and all of them actually come from QosEvent. Can it have only one parameter?
```suggestion
        void LogTestCoverageRawData(QosEvent e);
        // or preferable using a struct:
        void LogTestCoverageRawData(TestCoverageRawData d);
        struct TestCoverageRawData
        {
          string ModuleName;
          ...
        }
```

Reply:
Please add inline docs as well

Reply:
The original reason is AzurePSQosEvent is defined in the Common project and Abstraction project does not have reference of the Common. Otherwise, it requires to add more project references. I will try to move the interface to the Common project and then this can be resolved.

Reply:
Moved and replaced the parameters with AzurePSQosEvent